### PR TITLE
Set up users#show action

### PR DESCRIPTION
**Pull request by dsimp**

### CODE REVIEW

File: app/views/users/show.html.erb
Comment (jessjno) [inline app/views/users/show.html.erb:1]: nice formatting!

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<h1>
+  <%= @user.username %>
+</h1>
+
+<h2>Own photos</h2>
+
+<ul>
+  <% @user.own_photos.each do |photo| %>
+    <div class="row mb-4">
+      <div class="col-md-6 offset-md-3">
```
Comment (jessjno) [inline app/views/users/show.html.erb:10]: nice job not forgetting that first '-' in col-md-5

File: app/controllers/users_controller.rb
Comment (Testell) [inline app/controllers/users_controller.rb:1]: looks good

**Review by jessjno — APPROVED** [review]
looks good!

File: config/routes.rb
Comment (Testell) [inline config/routes.rb:1]: nice route!

**Review by Testell — APPROVED** [review]

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
```
Comment (marywebby) [inline app/controllers/comments_controller.rb:29]: good job implementing `redirect_back`!

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -11,22 +11,33 @@
     </div>
   <% end %>
 
-  <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
+  <%= form.hidden_field :photo_id %>
```
Comment (marywebby) [inline app/views/comments/_form.html.erb:14]: 👍

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,39 @@
+<h1>
+  <%= @user.username %>
+</h1>
+
+<h2>Own photos</h2>
+
+<ul>
+  <% @user.own_photos.each do |photo| %>
```
Comment (marywebby) [inline app/views/users/show.html.erb:8]: could also use current_user here but this works as well!

**Review by marywebby — APPROVED** [review]
Looks good! Nice work

**Pull request merged**